etl_manager/meta.py

```diff
@@ -234,26 +201,54 @@ def to_dict(self) :
             "location" : self.location
         }
         return meta
-    
+
     def write_to_json(self, file_path) :
         _write_json(self.to_dict(), file_path)
 
+    def refresh_paritions(self, temp_athena_staging_dir = None, database_name = None):
+        """
+        Refresh the partitions in a table, if they exist
+        """
+
+        if self.partitions:
+            if not temp_athena_staging_dir:
+                if self.database:
+                    temp_athena_staging_dir = self.database.s3_athena_temp_folder
+                else:
+                    raise ValueError("You must provide a path to a directory in s3 for Athena to cache query results")
+
+            conn = connect(s3_staging_dir = temp_athena_staging_dir, region_name = 'eu-west-1')
+
+            if not database_name:
+                if self.database:
+                    database_name = self.database.name
+                else:
+                    raise KeyError("You must provide a database name, or register a database object against the table")
+
+            sql = "MSCK REPAIR TABLE {}.{}".format(database_name, self.name)
+
+            try:
+                with conn.cursor() as cursor:
+                    cursor.execute(sql)
+            finally:
+                conn.close()
+
 
 class DatabaseMeta :
     """
-    Python class to manage glue databases from our agnostic meta data. 
+    Python class to manage glue databases from our agnostic meta data.
     db = DatabaseMeta('path_to_local_meta_data_folder/')
     This will create a database object that also holds table objects for each table json in the folder it is pointed to.
     The meta data folder used to initialise the database must contain a database.json file.
     """
     def __init__(self, database_folder_path, db_suffix = '_dev') :
-        
+
         self._tables = []
         database_folder_path = _end_with_slash(database_folder_path)
-        
+
         db_meta = _read_json(database_folder_path + 'database.json')
 
-        self.name = db_meta['name']
+        self.name = db_meta['name'] + db_suffix
```

Comment:
Thoughts about changing this to base_folder? E.g. for crest default would be `crest_v1_dev` and deployed would be `crest_v1`?